docs/admin/config/batch_changes.mdx

```diff
@@ -344,12 +344,37 @@ However, if the user deletion is permanent, deleting both account and data, then
 
 ## Batch Spec Library
 
-<Callout type="note"> Batch Spec Library is currently in Experimental.</Callout>
+The Batch Spec Library accelerates batch change adoption by offering curated examples and guided templates that make large-scale code modifications accessible to developers at every skill level.
 
-The Batch Spec Library is a collection of Batch Specs that can be used to create Batch Changes. Sourcegraph provides a few Batch Specs out of the box.
+The library distinguishes between [templates](#templates) and [library examples](#library-examples), where library examples are complete batch specs and templates are library examples with variables.
```

Comment:
I'd just say "examples" (not "library examples") here, since you say library @ the start of the sentence.

Reply:
This sentence strikes me as a little bit confusing: 

> where library examples are complete batch specs and templates are library examples > with variables.

Maybe we could rephrase to something like "[new sentence] Examples are batch specs (link to specs documentation) that are meant to be modified by power users who are comfortable writing code; Templates, on the other hand, are batch specs that contain variables that the user can inject data dynamically into (e.g. a form), and are meant to be used by general users."